File: samples/conference/management-bc/src/main/java/com/microsoft/conference/management/domain/Events/SeatTypeAdded.cs
using System;

namespace ConferenceManagement
{
    public class SeatTypeAdded : SeatTypeEvent
    {
        public int Quantity { get; private set; }

        public SeatTypeAdded() { }
        public SeatTypeAdded(Guid seatTypeId, SeatTypeInfo seatTypeInfo, int quantity)
            : base(seatTypeId, seatTypeInfo)
        {
            Quantity = quantity;
        }
    }
}

File: samples/conference/management-bc/src/main/java/com/microsoft/conference/management/domain/Events/SeatTypeAdded.java
package com.microsoft.conference.management.domain.Events;

import com.microsoft.conference.management.domain.Models.SeatTypeInfo;

public class SeatTypeAdded extends SeatTypeEvent {
    public int Quantity;

    public SeatTypeAdded() {
    }

    public SeatTypeAdded(String seatTypeId, SeatTypeInfo seatTypeInfo, int quantity) {
        super(seatTypeId, seatTypeInfo);
        Quantity = quantity;
    }
}
